src/components/batch-selection/batch-selection.component.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import I18n from 'i18n-js';
+import PropTypes from 'prop-types';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import { StyledBatchSelection, StyledSelectionCount } from './batch-selection.style';
+
+const BatchSelection = ({
+  disabled,
+  children,
+  colorTheme,
+  selectedCount,
+  hidden
+}) => {
+  return (
+    <StyledBatchSelection
+      colorTheme={ colorTheme }
+      data-component='batch-selection'
+      disabled={ disabled }
+      hidden={ hidden }
+    >
+      <StyledSelectionCount data-element='selection-count'>
+        <span>{ selectedCount }</span> { I18n.t('selected') }
```

Comment:
I think this should be namespaced.
```suggestion
        <span>{ selectedCount }</span> { I18n.t('batch-selection.selected') }
```

Reply:
good point, changed